### PR TITLE
Fixed bug in multitarget macenko during import

### PR DESCRIPTION
**Pull request by andreped**
Doing this was producing an error:
```
python -c "import torchstain; torchstain.normalizers.MultiMacenkoNormalizer(backend='torch')"
```

But after these changes, it is possible to use as with any other normalizer.

However, I am struggling to understand how to use it properly, as it just seem to crash all the time, but that can be handled in a separate issue/PR.

### CODE REVIEW

**Comment by andreped** [comment]
Any thoughts on this @desi-ivanov? Do you have a small code snippet showing how to use it? Currently, I just get an error when I use it on simple example images.

**Comment by carloalbertobarbano** [comment]
I'll merge the PR and take a look at the bug
